test/integration/admin_login_test.rb

```diff
@@ -3,30 +3,33 @@
 class AdminUserTest < ActionDispatch::IntegrationTest
   include Capybara::DSL
 
-  attr_reader :user, :project1, :project2, :category1, :category2
-
-  def setup
-    @user = User.create(username: "user",
-                        password: "password",
-                        first_name: "John",
-                        last_name: "Doe",
-                        email: "example@example.com"
-                       )
-    @category1 = Category.create(name: "Hot Beverages")
-    @category2 = Category.create(name: "cold beverages")
-    @project1 = category1.projects.create(title: "espresso",
-                                          description: "this is black gold",
-                                          price: 30000)
-    @project2 = category2.projects.create(title: "cold pressed coffee",
-                                          price: 8000,
-                                          description: "hipster nonsense",
-                                          price: 20000)
+  test "an admin can log in and get to the dashboard" do
+    create(:admin)
+
+    visit root_path
+    fill_in "session[username]", with: "admin"
+    fill_in "session[password]", with: "password"
+    click_link_or_button("Login")
+
+    assert page.has_content?("Platform Admin")
   end
 
-  test "an admin user can view home page" do
-    skip
+  test "an admin when logged in has an Admin Dashboard link" do
```

Comment:
This is a test to see what it looks like if the admin does visit the home page. The admin is redirected to platform dashboard by default.